src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/RecordComponentNumberCheckTest.java

```diff
@@ -153,4 +155,20 @@ public void testRecordComponentNumberPrivateModifier() throws Exception {
         verifyWithInlineConfigParser(
                 getNonCompilablePath("InputRecordComponentNumberPrivateModifier.java"), expected);
     }
+
+    @Test
+    public void testSetAccessModifiers() throws Exception {
```

Comment:
we need details on what this exactly is covering, code is changing it might be unrelated test soon.
We agreed to put javadoc explanation on reasoning of pure unit test.

may be copy to pitest message ?

Reply:
should we create util method for clone to let us ease skip/suppression of mutation on it:
```
    <description>replaced call to java/util/Arrays::copyOf with argument</description>
    <lineContent>Arrays.copyOf(accessModifiers, accessModifiers.length);</lineContent>
```

we do bunch of copy of content for over-security and fanatism of best practices. Such tests has no quality value, so we can make a code that should be skipped by pitest.

Reply:
or it will be just another junit method in all Test classes that check that clone is done for not simple type, as we have test methods for tokens.

Reply:
Done.

Reply:
> should we create util method for clone to let us ease skip/suppression of mutation on it:

A util for production code, will end up in the same situation as `Arrays.copyOf` is also a util method.
A util for test code seems fine to me, but it is really not a lot of code involved

Reply:
Please rename to `testCloneInAccessModifiersProperty()`

Reply:
Done.